jujugui/static/gui/src/app/components/deployment/summary/summary.js

```diff
@@ -43,6 +43,55 @@ YUI.add('deployment-summary', function() {
       validateForm: React.PropTypes.func.isRequired
     },
 
+    getInitialState: function() {
+      return {
+        regions: [],
+        activeRegion: this.props.deploymentStorage.region
+      };
+    },
+
+    componentWillMount: function() {
+      // If the model has already been deployed then we already
+      // have all the information we need.
+      var props = this.props;
+      if (props.modelCommitted) { return; }
```

Comment:
This might need a comment to remind us why we don't need to get the region info in this case.